utmap-client/src/components/pages/CalendarPage.js

```diff
@@ -80,6 +106,7 @@ const groupEvents = (eventsList) => {
 	//Groups events on the same day together if there are more than 3
 	let count = 0;
 	return eventsList.reduce((newList, event) => {
+		event = convertToCalendarEvent(event);
 		let lastElem = newList[newList.length - 1];
 		if(newList.length === 0 || 
 			!isSameDay(Date.parse(event.startDate), Date.parse(lastElem.startDate))) {
```

Comment:
Need to change it here

Reply:
Just to clarify, do you mean to convert `event` back to a UTMap event? Unless I'm misunderstanding!

Reply:
Oops, I commented on the wrong line. See line 84/112, event.startDate